src/timer-bar-entity-row.ts

```diff
@@ -17,6 +17,7 @@ export function fillConfig(config: TimerBarEntityConfig): TimerBarConfig {
     end_time: { attribute: 'end_time' },
     start_time: { attribute: 'start_time' },
     duration: { attribute: 'duration' },
+    remain_time: { attribute: 'remain_time' },
```

Comment:
`remain_time` seems a little lengthy of a name. What do you think of `remainder`?